users/User.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose')
+const bcrypt = require('bcrypt')
+const ObjectId = mongoose.Schema.Types.ObjectId
+
+const userSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+    lowercase: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
+  note: {
+    type: ObjectId,
+    ref: 'notes'
+  }
+})
```

Comment:
Faris, Its generally a good practice to add createdAt and modifiedAt field in a schema to track changes